app/views/feedback/new.html.erb

```diff
@@ -1,44 +1,57 @@
-<%= title "Submit Feedback" %>
+<%= title "Help" %>
```

Comment:
I agree that that would look slightly nicer, but it makes the code less readable. We would have a /help route that uses a FeedbackController and creates Feedback objects. I'm going to leave this alone for now.